### ISSUE TITLE
Problem with architect.js 

### ISSUE THREAD

**Issue by lehuydev2203** [brave-digital/react-native-wikitude#12]
I have problem with link https://wikitude.com/libs/architect.js . It's not found .

![image](https://user-images.githubusercontent.com/41155784/48791909-79edca80-ed25-11e8-89df-ca41b36f307c.png)

and My code : 
![image](https://user-images.githubusercontent.com/41155784/48792224-4bbcba80-ed26-11e8-994d-dc4b368ca82b.png)

**Comment by JanOwiesniak** [comment]
I think this issue is unrelated to react-native-wikitude. It looks like this is expected behaviour.

![not_found](https://user-images.githubusercontent.com/213214/48793063-1c8e5500-ecf6-11e8-9d3f-7b126eb5d303.png)

http://www.wikitude.com/external/doc/documentation/latest/android/samples.html

![expected](https://user-images.githubusercontent.com/213214/48793111-3fb90480-ecf6-11e8-8521-61bdce0bd9e7.png)

https://support.wikitude.com/support/discussions/topics/5000086748

**Comment by lehuydev2203** [comment]
Yes , i know it , i tried to try it . but It seems that things are not very favorable .

**Comment by JanOwiesniak** [comment]
First you should figure out if you ar scene is broken or not.

One possible way of doing this would be to [download the iOS or Android SDK](https://www.wikitude.com/download/) and run a [sample scene](https://github.com/Wikitude/wikitude-sdk-samples). If this is working, replace the sample scene with your scene.

Another way would be to run a sample scene in your current react-native-wikitude stack.

**Comment by lehuydev2203** [comment]
ok , thanks you , i will try

**Comment by thylo** [comment]
@JanOwiesniak Just to be sure, when you talk about the "ios sdk" you do mean the "IOS Javascript SDK" ?

**Comment by JanOwiesniak** [comment]
Yes, i’m talking about the Javascript SDK (available for iOS and Android)
